### DISCUSSION TITLE
Easier LUA access?

### DISCUSSION THREAD

**Discussion by Jonathan-Ott** [EdgeTX/edgetx#3991] [Q&A]
I asked this question on the EdgeTX Users FB group about this time last year, but never asked it here, so here goes:

Way back when I had a FrSky QX7, you could push one of the buttons (page down or up?) to screen over to the Betaflight LUA script and it would activate and you could do your Betaflight stuff (change PIDs, change vTx freq's, etc.). I know we have Betaflight LUA scripts for the Radiomaster TX16S and have to do a fair amount of button pushing to run the script, and have to do it again if you need to change something after you have turned the transmitter off...is there a way to get to the LUA scripts easier, like creating a widget page or something so it's just a couple clicks of the page button or such to launch the script?

Does this make sense?

**Comment by Jonathan-Ott** [comment]
I didn't think I asked this question here, but I did. This one is closed.

**Reply by pfeerick** [reply]
Can you add a cross-reference to the other place you raised this then? I was thinking I'd seen the suggestion for this before but can't place a finger on where that was now. Anyway, at the risk of repeating myself, I think the resulting idea was perhaps "shortcut widgets" that would let you start standalone lua?

**Reply by Jonathan-Ott** [reply]
@pfeerick @raphaelcoeffic

@pfeerick

Peter, this is the link to the previous posting: https://github.com/EdgeTX/edgetx/discussions/2404

I think a shortcut widget might be the answer. This [YouTube video](https://www.youtube.com/watch?v=zZq9JpHpGnk) is what I did (not my video) to get to the Betaflight script easier. Is this something we could incorporate into EdgeTX?

Thanks!

Jon
